app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>WwwZaznovaCom</title>
+  <title>Zaznova</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
+  </script>
 </head>
 <body>
 
-<%= yield %>
+  <%= yield %>
 
 </body>
+<%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```

Comment:
I think turbolinks should be removed. It causes conflicts with jquery events. There are some tricks to avoid those issues but in think we don't need turbolinks for now.